### ISSUE TITLE
Make sure the code is `dask`-friendly

### ISSUE THREAD

**Issue by malmans2** [pyNEMO/pyDOMCFG#20]
After #19, add a separate test environment to test with `dask`.
We'd have to introduce a check to make sure that nothing triggers eager evaluation and we return a lazy dataset.

(Also, would be nice to have an idea of performace with an input Bathymetry file such as eORCA36)
